pack/side/blbe_encounter.json

```diff
@@ -1,4 +1,422 @@
+    "name": "Replicating Aberration",
+    "pack_code": "blbe",
+    "position": 10,
+    "quantity": 1,
+    "text": "<b>Replicate</b> - If you spot an investigator at a location with a copy of Sticky Feet attached to it, spawn this enemy exhausted at that location (unless the investigators spend a countermeasure).\nHunter.\n<b>Forced</b> - When Replicating Aberration leaves play: Return it to the event organizer.\n<b>Blob 3.</b>",
+    "traits": "Monster. Ooze. Manifold.",
+    "type_code": "enemy"
+  },
+
+  {
+    "back_text": "<b>Read only if \"the escort failed:\"</b>\nRemove Mi-Go Destroyer, Armored Car, and this card from the game. Each investigator takes 1 damage. Lose 1 countermeasure.\n<hr><b>Read only if \"the car reached its target:\"</b>\n<blockquote><i>It’s chilling what a machine of war can do when it’s fully unleashed, but in this case, you’re relieved at the thundering of its gun and the crunch of its tires on alien ground. With this area secure, the military can focus its efforts on the anomaly itself. Although they don’t thank you for your efforts directly, the detachment of agents following you around tells you they view you as an asset.</i></blockquote>\nPut the set-aside G-Men story asset into play under any investigator’s control. Remove Armored Car from the game. Add Mi-Go Destroyer and this card to the victory display. Deal 5 damage to Subject 8L-08.",
+    "back_traits": "Part 2.",
+    "code": "89011",
+    "double_sided": true,
+    "encounter_code": "migo_incursion_2",
+    "encounter_position": 1,
+    "faction_code": "mythos",
+    "flavor": "You’ve heard from some of the civilians fleeing to the perimeter of the quarantine zone that the military is sending in armor to secure Blackwater. You’re not sure a bigger gun is the answer, but it might help deter the creatures that have been wreaking havoc across the town. If you can clear the path and prevent it from getting devoured like the rest of the vehicles they’ve sent in, maybe it can make a real difference.",
+    "name": "Escort the Car",
+    "pack_code": "blbe",
+    "position": 11,
+    "quantity": 1,
+    "text": "<b>Revelation</b> - Put the set-aside Armored Car into play at the location farthest from the Fungus Mound. Spawn the set-aside Mi-Go Destroyer enemy at the Fungus Mound. Put this card into play next to the act deck.\n<b>Forced</b> - After Armored Car enters the Fungus Mound: Remember that \"the car reached its target.\" Flip this card over.\n<b>Forced</b> - If Armored Car is defeated or devoured: Remember that \"the escort failed.\" Flip this card over.",
+    "traits": "Part 1.",
+    "type_code": "story",
+    "victory": 1
+  },
+
+  {
+    "code": "89012",
+    "cost": null,
+    "encounter_code": "migo_incursion_2",
+    "encounter_position": 2,
+    "faction_code": "neutral",
+    "health": 5,
+    "illustrator": "Adam Lane",
+    "name": "Armored Car",
+    "pack_code": "blbe",
+    "position": 12,
+    "quantity": 1,
+    "sanity": null,
```

Comment:
For some reason historically for blank health/sanity values we've simply skipped the line, but putting null in here should be fine too.